.github/workflows/performance_testing_manual.yml

```diff
@@ -0,0 +1,39 @@
+name: Benchmark Report generation
+
+on:
+  workflow_dispatch:
+    inputs:
+      rounds:
```

Comment:
Nitpick: maybe call this `iterations` or `test_iterations` rounds is a little confused wording here.

Reply:
That would be something different. Rounds and iterations are treated differently in pytest-benchmark and as such applying your suggestion would be incorrect.

See: https://pytest-benchmark.readthedocs.io/en/latest/pedantic.html#benchmark.pedantic

Reply:
Ah interesting, nevermind then.

Reply:
Since our methods are all slower then 100 microseconds we are fine with an iteration size of one.